src/main/flight/gps_rescue.c

```diff
@@ -453,7 +456,7 @@ static void performSanityChecks(void)
     if (rescueState.phase == RESCUE_FLY_HOME) {
         const float velocityToHomeCmS = previousDistanceToHomeCm - rescueState.sensor.distanceToHomeCm; // cm/s
         previousDistanceToHomeCm = rescueState.sensor.distanceToHomeCm;
-        rescueState.intent.secondsFailing += (velocityToHomeCmS < 0.5f * rescueState.intent.targetVelocityCmS) ? 1 : -1;
+        rescueState.intent.secondsFailing += (velocityToHomeCmS < 0.1f * rescueState.intent.targetVelocityCmS) ? 1 : -1;
```

Comment:
Is this tested? 10% margin feels quite small ...

Reply:
It's still bringing the quad home :-)
The argument here is that so long as we are heading towards home, we should reduce the counter that brings us closer to disarming, even if we only make 10% of the requested headway, it's better than nothing.
In practicality, most rescues in 'normal' situation achieve very close to requested return velocities.
However, there are some unusual situations where this may not be the case.  For instance, if someone configured a fast return rate, but stupidly set a low value for the maximum pitch angle, or lacked adequate max throttle, and then encountered a headwind, their return to home velocity as a percentage of the set return velocity could be quite low, even though they could be making steady headway.
If they fail to make any headway (stuck in tree) or if they are consistently moving negative (away) then the sanity check will still work well.